package_test.go

```diff
@@ -364,3 +398,28 @@ const exampleDocumentsCreateResponse_1documents string = `{
 		"deletedAt": "2019-08-24T14:15:22Z"
 	}
 }`
+
+const exampleDocumentsUpdateResponse_1documents string = `{
```

Comment:
This is a normal `document` right?
Why not re-using the existing fake `document`?

Reply:
True, thanks :+1: